examples/hpc/dla-driver/src/lib.rs

```diff
@@ -16,9 +21,9 @@ pub use mmap::{
 };
 
 const DEFAULT_INPUT_BANK: MemoryBank = MemoryBank::Bank0;
-const DEFAULT_KERNEL_BANK: MemoryBank = MemoryBank::Bank8;
-const DEFAULT_OUTPUT_BANK: MemoryBank = MemoryBank::Bank12;
-const DEFAULT_BIAS_ADDR: u32 = 0x0;
+const DEFAULT_KERNEL_BANK: MemoryBank = MemoryBank::Bank4;
+const DEFAULT_OUTPUT_BANK: MemoryBank = MemoryBank::Bank10;
+const DEFAULT_BIAS_ADDR: u32 = (MEMORY_BANK_BASE_ADDR + MEMORY_BANK_15_OFFSET) as u32;
```

Comment:
This looks like it should be a bank as well. Isn't `DEFAULT_BIAS_ADDR` effectively set to `Bank15` now? In case you need to retrieve the address from a `MemoryBank` instance, consider adding/using a method like `addr()` on `MemoryBank`.

Reply:
Modified to use `addr()` to retrieve `MemoryBank` address for setting default bias